### ISSUE TITLE
Select accessiblity issues for JAWs and NVDA

### ISSUE THREAD

**Issue by jquense** [JedWatson/react-select#2937]
Great job ya'll with the recent efforts to make react-select more accessible. It performs great in VoiceOver! Unfortunately the strategy doesn't seem to work for the two most commonly used SR's JAWs and NVDA. 

For both, the live region alerts are not read aloud when you enter or select items in the list. This ends being mostly a silent black box for these SR's, and the input doesn't seem usable :/

I will admit i'm not the best SR user, so some of the issue may be me, but i am not a complete novice either. I'm also trying to com up with a good a11y strategy for these sorts of inputs for the next iteration of react-widgets!

**Comment by bladey** [comment]
Hello -

In an effort to sustain the `react-select` project going forward, we're closing old issues.

We understand this might be inconvenient but in the best interest of supporting the broader community we have to direct our efforts towards the current major version.

If you aren't using the latest version of `react-select` please consider upgrading to see if it resolves any issues you're having.

However, if you feel this issue  is still relevant and you'd like us to review it - please leave a comment and we'll do our best to get back to you!

**Comment by jquense** [comment]
Still relevant last I checked and important :) 

Thanks!

**Comment by bladey** [comment]
Thanks @jquense!

**Comment by ebonow** [comment]
Greetings @jquense ,

Would you kindly verify if this this is still an issue for you in version 4.2.1?

I spent a bit of time refactoring the aria-live implementation and was sure to render the region on mount, but was not personally able to do any testing outside of VoiceOver.

**Comment by jquense** [comment]
@ebonow thanks for working on this, I unfortunately don't have time to verify this, but if you or anyone else wants to test with NVDA, Microsoft provides free VM's for testing stuff (like old internet explorer), In the past i've used them to test with NVDA from my Mac

**Comment by ebonow** [comment]
@jquense I've been using virtual box but it seems laggy so I may have to reorganize some things to get the space for Bootcamp. There are two developers who have begun taking on accessibility testing that did verify the changes work on JAWS and NVDA, but wanted to verify with you in case you had a different experience.

I'll mark this as resolved especially since the linked PR which would have closed this PR was implemented in a similar way.

**Comment by jquense** [comment]
awesome thanks for all the hard work 👍

**Comment by Watso196** [comment]
Hey there! Curious if there are any plans to address the issue listed here? My team is still seeing this issue in the React Select and are wondering if this non-live region content should still be included with the update to role="combobox"?
